bench/thor/routes.cc

```diff
@@ -11,6 +11,7 @@
 #include "sif/costfactory.h"
 #include "test.h"
 #include "thor/bidirectional_astar.h"
+#include "thor/timedep.h"
```

Comment:
can we revert this file? i know this was a long running pr with many iterations but thsi seems irrelevant now?

Reply:
I was using bench for seeing the impact of the change on timedep a-star implementations. We're no longer doing that in this PR now, so I could revert it. Although I feel its okay to test the timedep implementations in the bench tests for completeness.